site/content/en/guides/consumer/function/catalog/_index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Functions Catalog"
+linkTitle: "Functions Catalog"
+weight: 8
+type: docs
+description: >
+    Catalog of kpt functions.
+---
+
+<!---
+DO NOT EDIT. Generated by: "cd catalog; npm run gen-docs"
```

Comment:
Where's the package after this move? At very least fully specify the path, but ideally this should be part of a top-level command when generating all docs.

Reply:
The catalog package is at googlecontainertools.github.io/kpt/guides/consumer/function/catalog. I also included a list of where sdk docs ended up in the description.

Reply:
Done